### PR TITLE
Hamburger task

### PR DESCRIPTION
**Pull request by shalinee-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/import-excel/import-type-select/import-type-select.component.html [hunk 1]
```diff
@@ -1,3 +1,4 @@
+<hamburger-menu [pageHeading]="'Import'"></hamburger-menu>
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/import-excel/import-type-select/import-type-select.component.html:1]: Page heading should be Import Data

File: apps/web-giddh/src/app/import-excel/import-wizard/import-wizard.component.html [hunk 2]
```diff
@@ -1,3 +1,4 @@
+<hamburger-menu [pageHeading]="'Import'"></hamburger-menu>
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/import-excel/import-wizard/import-wizard.component.html:1]: Page heading should be Import Data

File: apps/web-giddh/src/app/ledger/ledger.component.html [hunk 3]
```diff
@@ -10,6 +10,7 @@
                 (click)="showExportLedgerModal()">Export</a></li>
     </ul>
 </div>
+<hamburger-menu [pageHeading]="(lc?.activeAccount?.name) ? (' Ledger - ' + lc?.activeAccount?.name) + ' A/c' : ''"></hamburger-menu>
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/ledger/ledger.component.html:13]: Remove space before Ledger

File: apps/web-giddh/src/app/manufacturing/report/mf.report.component.html [hunk 4]
```diff
@@ -1,4 +1,6 @@
 <section id="manufacture" class="clearfix">
+    <!-- <hamburger-menu [pageHeading]="'Mfg. Journal Register'"></hamburger-menu> -->
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/manufacturing/report/mf.report.component.html:2]: Check this.

File: apps/web-giddh/src/app/models/defaultMenus.ts [hunk 5]
```diff
@@ -1,239 +1,230 @@
...
-    { type: 'MENU', name: 'Sales Invoice', uniqueName: '/pages/proforma-invoice/invoice/sales' },
-    { type: 'MENU', name: 'New Purchase Bill', uniqueName: '/pages/proforma-invoice/invoice/purchase' },
-    { type: 'MENU', name: 'Cash Invoice ', uniqueName: '/pages/proforma-invoice/invoice/cash' },
-    { type: 'MENU', name: 'Proforma Invoice', uniqueName: '/pages/proforma-invoice/invoice/proformas' },
-    { type: 'MENU', name: 'Estimate (Beta)', uniqueName: '/pages/proforma-invoice/invoice/estimates' },
-    { type: 'MENU', name: 'New Credit Note', uniqueName: '/pages/proforma-invoice/invoice/credit note' },
-    { type: 'MENU', name: 'New Debit Note', uniqueName: '/pages/proforma-invoice/invoice/debit note' },
-    { type: 'MENU', name: 'Reports > Receipt (Beta)', uniqueName: '/pages/reports/receipt' }
+    { type: 'MENU', name: 'Sales Invoice', uniqueName: '/pages/proforma-invoice/invoice/sales', hasTabs: false },
+    { type: 'MENU', name: 'Purchase Record ', uniqueName: '/pages/proforma-invoice/invoice/purchase', hasTabs: false },
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/models/defaultMenus.ts:212]:
```suggestion
    { type: 'MENU', name: 'New Purchase Bill', uniqueName: '/pages/proforma-invoice/invoice/purchase', hasTabs: false },
```

File: apps/web-giddh/src/app/reports/components/sales-register-component/sales.register.component.html [hunk 6]
```diff
@@ -1,8 +1,39 @@
...
+<div class="relative">
+    <hamburger-menu [pageHeading]="''"></hamburger-menu>
+    <div class="report-breadcrum">
+        <ul class="list-head list-inline d-inline-block middle nav-report">
+            <li><a href="#">Report</a> <span class="d-inline-block pl-1"> ></span></li>
+            <li><a href="#">Sales Register</a> <span class="d-inline-block pl-1"> ></span></li>
+            <li>
+                <div class="select-style width-200 select-year">
+                    <select name="">
+                        <option value="">August 2017</option>
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/reports/components/sales-register-component/sales.register.component.html:11]: Why static text is added?

File: apps/web-giddh/src/app/settings/settings.component.html [hunk 7]
```diff
@@ -8,45 +8,90 @@
...
-                    <setting-profile #profileComponent></setting-profile>
-                    <span *ngIf="isCompanyProfileUpdated && (isUpdateCompanyInProgress$ | async)" class="auto_save" style="float: right;">
-                        <i class="icon-refresh mrR fs14 rotate"></i>Saving...
-                    </span>
-                    <span *ngIf="isCompanyProfileUpdated && !(isUpdateCompanyInProgress$ | async)" class="auto_save" style="float: right;">
-                        <i class="glyphicon glyphicon-check mrR fs14"></i>Saved
-                    </span>
+
+                <div *ngIf="activeTab === 'profile'" class="relative">
+                    <hamburger-menu></hamburger-menu>
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/settings/settings.component.html:34]: No page heading required?

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/reports/components/sales-register-component/sales.register.component.html (hunk 6 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/reports/components/sales-register-component/sales.register.component.html:11]: This is old code I have replaced only position

**Review by aditi-tiwari17 — APPROVED** [review]

File: apps/web-giddh/src/app/ledger/ledger.component.html [hunk 8]
```diff
@@ -10,7 +10,7 @@
                 (click)="showExportLedgerModal()">Export</a></li>
     </ul>
 </div>
-<hamburger-menu [pageHeading]="(lc?.activeAccount?.name) ? (' Ledger - ' + lc?.activeAccount?.name) + ' A/c' : ''"></hamburger-menu>
+<hamburger-menu [pageHeading]="(lc?.activeAccount?.name) ? ('Ledger -' + lc?.activeAccount?.name) + ' A/c' : ''"></hamburger-menu>
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/ledger/ledger.component.html:13]: Add space after -

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

**Review by rohit-walkover — APPROVED** [review]

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**